cmd/gitbase/command/server.go

```diff
@@ -28,13 +32,15 @@ const (
 
 // Server represents the `server` command of gitbase cli tool.
 type Server struct {
-	Verbose  bool     `short:"v" description:"Activates the verbose mode"`
-	Git      []string `short:"g" long:"git" description:"Path where the git repositories are located, multiple directories can be defined. Accepts globs."`
-	Siva     []string `long:"siva" description:"Path where the siva repositories are located, multiple directories can be defined. Accepts globs."`
-	Host     string   `short:"h" long:"host" default:"localhost" description:"Host where the server is going to listen"`
-	Port     int      `short:"p" long:"port" default:"3306" description:"Port where the server is going to listen"`
-	User     string   `short:"u" long:"user" default:"root" description:"User name used for connection"`
-	Password string   `short:"P" long:"password" default:"" description:"Password used for connection"`
+	Verbose   bool     `short:"v" description:"Activates the verbose mode"`
+	Git       []string `short:"g" long:"git" description:"Path where the git repositories are located, multiple directories can be defined. Accepts globs."`
+	Siva      []string `long:"siva" description:"Path where the siva repositories are located, multiple directories can be defined. Accepts globs."`
+	Host      string   `short:"h" long:"host" default:"localhost" description:"Host where the server is going to listen"`
+	Port      int      `short:"p" long:"port" default:"3306" description:"Port where the server is going to listen"`
+	User      string   `short:"u" long:"user" default:"root" description:"User name used for connection"`
+	Password  string   `short:"P" long:"password" default:"" description:"Password used for connection"`
+	PilosaURL string   `long:"pilosa" default:"http://localhost:10101" description:"URL to your pilosa server"`
+	IndexDir  string   `short:"i" long:"index" default:"/var/gitbase/index" description:"Directory where the gitbase indexes information will be persisted."`
```

Comment:
Maybe we should change the default path. What do you think @jfontan ?

Reply:
What path should we change it to?

Reply:
Usually software does not have var directories directly in `/var` but in one of its subdirectories. MySQL has its database files in `/var/lib/mysql`.